packages/shared/lib/core/wallet/classes/activity.class.ts

```diff
@@ -79,7 +79,7 @@ export class Activity implements IActivity {
         const isInternal = !!findAccountWithAddress(address)
         this.id = outputId
         this.outputId = outputId
-        this.time = new Date(output.metadata.milestoneTimestampBooked)
+        this.time = new Date(output.metadata.milestoneTimestampBooked * 1000)
```

Comment:
There is some constants already in the time lib so we don't need to use magic numbers and make tis more readable. i.e. SECOUNDS_PER_MINUTE